src/Mvc/Mvc.Core/src/ModelBinding/Metadata/DefaultMetadataDetails.cs

```diff
@@ -64,6 +69,11 @@ public DefaultMetadataDetails(ModelMetadataIdentity key, ModelAttributes attribu
         /// </summary>
         public Action<object, object> PropertySetter { get; set; }
 
+        /// <summary>
+        /// Gets or sets a delegate used to construct an object using the model binding constructor.
```

Comment:
What is a "model binding constructor"❔ I think you mean something like "bound constructor for <c>record</c> types"